RecoTracker/FinalTrackSelectors/src/TrackMVAClassifierBase.cc

```diff
@@ -65,12 +67,17 @@ void TrackMVAClassifierBase::produce(edm::Event& evt, const edm::EventSetup& es
   }
 
   // products
-  auto mvas = std::make_unique<MVACollection>(tracks.size(),-99.f);
+  auto mvas  = std::make_unique<MVACollection>(tracks.size(),-99.f);
   auto quals = std::make_unique<QualityMaskCollection>(tracks.size(),0);
 
-  
-  
-  computeMVA(tracks,*hBsp,*hVtx,forest,*mvas);
+  if ( hVtx.isValid() && !ignoreVertices_ ) {
+    computeMVA(tracks,*hBsp,*hVtx,forest,*mvas);
+  } else {
+    if ( !ignoreVertices_ ) 
+      edm::LogWarning("TrackMVAClassifierBase") << "ignoreVertices is set to False in the configuration, but the vertex collection is not valid"; 
```

Comment:
hi @mtosi is the log warning meant to indicate  a "ok" configuration or a job misconfiguration?